rules/windows/discovery_net_command_system_account.toml

```diff
@@ -1,7 +1,8 @@
 [metadata]
 creation_date = "2020/03/18"
-maturity = "production"
-updated_date = "2021/03/03"
+deprecation_date = "2021/04/15"
+maturity = "deprecated"
```

Comment:
if we remove net.exe from this rule is it still noisy?

Reply:
This one isn't noisy removing `net.exe`, but we have a limited windows testbed. I think this could be expanded with a sequence. I'll remove it from deprecation remove `net.exe` from the rule in https://github.com/elastic/detection-rules/pull/1114/ for now.